### PR TITLE
SQLite in dev, SS in production links

### LINKED ISSUE #21609: Fix links to Use SQLite for development, SQL Server for production
Links need to specify the VSC tab

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes #21609

Test all links once we have a build.

### CODE REVIEW

File: aspnetcore/tutorials/first-mvc-app/adding-model.md [hunk 1]
```diff
@@ -411,6 +413,7 @@ Because the `Model` object is strongly typed (as an `IEnumerable<Movie>` object)
 
 ## Additional resources
 
+* [Use SQLite for development, SQL Server for production](?tabs=visual-studio-code#sqlite-dev)
```
Comment (Rick-Anderson) [inline aspnetcore/tutorials/first-mvc-app/adding-model.md:416]: @scottaddie  will this in page link work. With the default tab, need to specify the VSC tab.

**Comment by scottaddie** [comment]
@Rick-Anderson Shouldn't the _aspnetcore/includes/RP/sqlitedev.md_ include replace the following highlighted section in _aspnetcore/tutorials/razor-pages/model.md_?
image

File: aspnetcore/includes/RP/sqlitedev.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ---
 no-loc: [appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
 ---
+<a name="sqlite-dev"></a>
 ### Use SQLite for development, SQL Server for production
 
 When SQLite is selected, the template generated code is ready for development. The following code shows how to inject <xref:Microsoft.AspNetCore.Hosting.IWebHostEnvironment> into Startup. `IWebHostEnvironment` is injected so `ConfigureServices` can use SQLite in development and SQL Server in production.
```
Comment (scottaddie) [inline aspnetcore/includes/RP/sqlitedev.md:7]:
```suggestion
When SQLite is selected, the template generated code is ready for development. The following code shows how to inject <xref:Microsoft.AspNetCore.Hosting.IWebHostEnvironment> into `Startup`. `IWebHostEnvironment` is injected so `ConfigureServices` can use SQLite in development and SQL Server in production.
```

File: aspnetcore/tutorials/razor-pages/model.md [hunk 3]
```diff
@@ -364,6 +366,8 @@ The next tutorial explains the files created by scaffolding.
 
 ## Additional resources
 
+[Use SQLite for development, SQL Server for production](?tabs=visual-studio-code#sqlite-dev)
```
Comment (scottaddie) [inline aspnetcore/tutorials/razor-pages/model.md:369]: Let's put the querystring at the end. I think that will still work.

```suggestion
[Use SQLite for development, SQL Server for production](#sqlite-dev?tabs=visual-studio-code)
```

File: aspnetcore/tutorials/first-mvc-app/adding-model.md (hunk 1 above)
Comment (scottaddie) [reply aspnetcore/tutorials/first-mvc-app/adding-model.md:416]: See my suggestion about putting the querystring at the end.

**Review by scottaddie — CHANGES_REQUESTED** [review]

File: aspnetcore/tutorials/first-mvc-app/adding-model.md (hunk 1 above)
Comment (Rick-Anderson) [inline aspnetcore/tutorials/first-mvc-app/adding-model.md:416]:
```suggestion
[Use SQLite for development, SQL Server for production](#sqlite-dev?tabs=visual-studio-code)
```

File: aspnetcore/tutorials/first-mvc-app/adding-model.md [hunk 4]
```diff
@@ -411,6 +413,7 @@ Because the `Model` object is strongly typed (as an `IEnumerable<Movie>` object)
 
 ## Additional resources
 
+[Use SQLite for development, SQL Server for production](#sqlite-dev?tabs=visual-studio-code)
```
Comment (scottaddie) [inline aspnetcore/tutorials/first-mvc-app/adding-model.md:416]:
```suggestion
* [Use SQLite for development, SQL Server for production](#sqlite-dev?tabs=visual-studio-code)
```

**Review by scottaddie — APPROVED** [review]
This looks good after incorporating the following suggestion.

File: aspnetcore/tutorials/razor-pages/model.md [hunk 5]
```diff
@@ -364,6 +356,8 @@ The next tutorial explains the files created by scaffolding.
 
 ## Additional resources
 
+[Use SQLite for development, SQL Server for production](#sqlite-dev?tabs=visual-studio-code)
```
Comment (Rick-Anderson) [inline aspnetcore/tutorials/razor-pages/model.md:359]:
```suggestion
```

**Pull request merged**